login-workflow/src/new-architecure/components/workflow/WorkflowCardHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import CardHeader from '@mui/material/CardHeader';
+import Typography from '@mui/material/Typography';
+import { WorkflowCardHeaderProps } from './WorkflowCard.types';
+
+export const WorkflowCardHeader: React.FC<WorkflowCardHeaderProps> = (props) => {
+    const { title, sx, ...cardHeaderProps } = props;
+
+    return (
+        <>
+            <CardHeader
+                title={<Typography variant={'h6'}>{title}</Typography>}
```

Comment:
If we go with the default type defined for title in CardHeaderProps, then we should define this as:
```suggestion
                title={typeof title === 'string' ? <Typography variant={'h6'}>{title}</Typography> : title}
```